libbeat/scripts/generate_fields_docs.py

```diff
@@ -70,8 +70,21 @@ def fields_to_asciidoc(input, output, beat):
 
 
     docs = yaml.load(input)
+
+    # fields file is empty
+    if docs is None:
+        return
```

Comment:
This mainly is related to beat-generator. The above case for example if fields.yml is empty (which I think should be fine for the moment). But I like the idea that we transform this to readable "information" which is printed what people should do to improve it. Would just using "print" here be fine?